core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -1060,9 +1075,7 @@ public Builder removeSnapshots(Collection<Long> idsToRemove) {
           danglingRefs.add(refEntry.getKey());
         }
       }
-
-      danglingRefs.forEach(this::removeBranch);
-
+      danglingRefs.forEach(this::removeRef);
```

Comment:
Can you revert the unnecessary whitespace changes here?

Reply:
Updated!=